Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -87,21 +89,27 @@
 
     private SharedPreferences mSharedPreferences;
     private String mToken;
-    private DrawerLayout mDrawer;
+
     private Handler mHandler;
-    private NavigationView mNavigationView;
     private int mPreviousMenuItemId;
     private static final String travelShortcut = "io.github.project_travel_mate.TravelShortcut";
     private static final String myTripsShortcut = "io.github.project_travel_mate.MyTripsShortcut";
     private static final String utilitiesShortcut = "io.github.project_travel_mate.UtilitiesShortcut";
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+    @BindView(R.id.drawer_layout)
+    DrawerLayout mDrawer;
```

Comment:
if variables are not private, no need to follow this naming convention.
Remove `m` from start

Reply:
`drawerLayout`